src/plugins/vis_type_timeseries/public/application/components/aggs/math.js

```diff
@@ -114,12 +114,12 @@ export function MathAgg(props) {
                 values={{
                   link: (
                     <EuiLink
-                      href="https://github.com/elastic/tinymath/blob/master/docs/functions.md"
+                      href="https://github.com/elastic/kibana/blob/master/packages/kbn-tinymath/docs/functions.md"
                       target="_blank"
                     >
                       <FormattedMessage
                         id="visTypeTimeseries.math.expressionDescription.tinyMathLinkText"
-                        defaultMessage="TinyMath"
+                        defaultMessage="@kbn/tinymath"
```

Comment:
Do we want to link to it under this technical name? I think the old label was fine for user-facing communication